packages/ripple-binary-codec/src/index.ts

```diff
@@ -112,4 +134,8 @@ export {
   decodeQuality,
   decodeLedgerData,
   TRANSACTION_TYPES,
+  XrplDefinitions,
+  XrplDefinitionsBase,
```

Comment:
Shouldn't this class not be exported, since we just want people to use `XrplDefinitions`?